crates/sncast/tests/e2e/account/add.rs

```diff
@@ -194,6 +209,40 @@ pub async fn test_nonexistent_account_address() {
     "});
 }
 
+#[tokio::test]
+pub async fn test_nonexistent_account_type() {
+    let tempdir = tempdir().expect("Unable to create a temporary directory");
+    let accounts_file = "accounts.json";
+
+    let args = vec![
+        "--url",
+        URL,
+        "--accounts-file",
+        accounts_file,
+        "account",
+        "add",
+        "--name",
+        "my_account_add",
+        "--address",
+        "0x123",
+        "--private-key",
+        "0x456",
+        "--type",
+        "unknown_type",
+    ];
+
+    let snapbox = runner(&args).current_dir(tempdir.path());
+    let output = snapbox.assert().failure();
+
+    assert_stderr_contains(
+        output,
+        indoc! {r"
+        error: invalid value 'unknown_type' for '--type <ACCOUNT_TYPE>'
+          [possible values: oz, argent]
+    "},
+    );
+}
```

Comment:
This is logic of clap library, not sure if it is useful test in the context of cast. It is basically isomorphic to our listing in the command args.

Reply:
It serves as a kind of sanity check. Personally I think that it makes some sense for e2e test, but I'm also fine with removing it.